src/components/Tabs/utils.ts

```diff
@@ -10,22 +11,27 @@ export const getTabContainerStyle = (
   itemIndex: number,
   itemRef: React.RefObject<HTMLDivElement>
 ) => {
-  let styles: React.CSSProperties = { zIndex: 99, position: "relative" }
+  let styles = `z-index: 99; position: relative;`
   if (activeKeyIndex === itemIndex) {
-    styles = { zIndex: 101 }
+    styles = `z-index: 101`
   }
   if (activeTabFlags.left && activeKeyIndex === itemIndex) {
-    styles = {
-      position: "sticky",
-      zIndex: 101,
-      left: 0,
-    }
+    styles = `
+      position: sticky;
+      position: -webkit-sticky;
+      z-index: 101;
+      left: 0;
+      align-self: flex-start;
```

Comment:
align-self, it's not required here.. as here flex-direction is row & it will align vertically.